scripts/west_commands/reset.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2018 Open Source Foundries Limited.
+# Copyright 2019 Foundries.io
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+'''west "reset" command'''
+
+from west.commands import WestCommand
+
+from run_common import add_parser_common, do_run_common
+
+
+class Reset(WestCommand):
+
+    def __init__(self):
+        super(Reset, self).__init__(
+            'reset',
+            # Keep this in sync with the string in west-commands.yml.
+            'reset and run a binary on a board',
+            "Permanently reprogram a board's reset with a new binary.",
```

Comment:
This text is wrong isn't it? This doesn't reprogram?

Reply:
Copied from the flash command, will fix that.